lib/toCamelCase.ts

```diff
@@ -0,0 +1,10 @@
+// Convert a CamelCase string to kebab-case
+export const toCamelCase = function toCamelCase(string = ""): string {
+  console.log({ string });
```

Comment:
We probably don’t want a console.log here

Reply:
fixed on commit 2db8f61b4f8dc8e492e9c541ae7bfb288cd17f76